src/hooks/useLiquidityMiningFeatureFlag.ts

```diff
@@ -3,5 +3,5 @@ import { useActiveWeb3React } from '.'
 
 export function useLiquidityMiningFeatureFlag(): boolean {
   const { chainId } = useActiveWeb3React()
-  return chainId === ChainId.RINKEBY || chainId === ChainId.XDAI
```

Comment:
Are we going to want to have XDAI on for the test on XDAI

Reply:
Definitely, added back now

Reply:
👍